discovery.md

```diff
@@ -523,9 +523,10 @@ as if that Service does not exist. For example, it would be excluded from
 any array of Services returned and it would result in a `404 Not Found`
 error for a request to that Service directly.
 
-Unless there is some out-of-band agreement, all APIs MUST use JSON encoding,
-which means when there is an HTTP Body the HTTP `Content-Type` Header MUST
-be `application/json`.
+All APIs MUST support JSON encoding, which means that an HTTP requests
+including an HTTP `Content-Type` Header of `application/json` and body in that
+format MUST be supported.  Likewise requests with `Accept` header of
```

Comment:
use of the phrase "MUST be supported" here (both of them) is a bit awkward - I think. To me, I'm reading it as "you MUST be able to accept them", but I think you're trying to say "when present, a Content-Type header MUST have a value of application/json when the Body is json", or something like that, no? Same for "Accept".
And it's app/json not app\\json

Reply:
No.  I was trying to maintain the original assertion.  The service can accept other types but it must accept JSON so when a request that declares it is JSON and actually contains valid JSON shows up, it must handle the request according to the specification.

Reply:
I'm apparently blind to the backwards slash/escape.  Where?

Reply:
Before resolving this because I do think it expresses a desired target (though, please do discuss if you disagree), I think there is a lot of room for better phrasing.  Basically I am attempting to declare that all compliant APIs must give JSON when requested and accept JSON when it is given.  However, I was trying to be more concrete than that in following suit with the previous declaration and being explicit about what "supporting JSON encoding" means.

Reply:
I"m ok with it - I think I was just misreading it before.